src/intl/en/learn-quizzes.json

```
@@ -324,5 +324,52 @@
   "h005-c-label": "Eth1",
   "h005-c-explanation": "Eth1 was the original name given to the execution layer, not the consensus layer.",
   "h005-d-label": "Staking",
-  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain."
+  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain.",
+  "i001-prompt": "What is required to 'run a node'?",
+  "i001-a-label": "Run software, with modest hardware, while online",
```

Wording here is weird. I'd drop 'while online' or change the wording somehow.

```suggestion
  "i001-a-label": "Running client software with modest hardware while staying online.",
```